antora.yml

```diff
@@ -8,6 +8,9 @@ asciidoc:
     bonitasoft-docker-repository: '{bonitasoft-registry}/docker'
     bonitaTechnicalVersion: '7.15.0' # latest public community version
     javadocVersion: 7.15
+    cscRootUrl: https://csc.bonitacloud.com/apps/CustomerServices
```

Comment:
**nitpick:** wrong URL, we will fix this with @ugaston

Reply:
Good catch!

Indeed, the good URL should read: csc.bonitacloud.**bonitasoft**.com

Thanks for the review.

Reply:
fixed in 62c07e0